sdk/servicebus/service-bus/test/streamingReceiver.spec.ts

```diff
@@ -1382,9 +1382,6 @@ describe("Streaming - disconnects", function(): void {
     // Simulate a disconnect being called with a non-retryable error.
     receiver["_context"].namespace.connection["_connection"].idle();
 
-    // Allow rhea to clear internal setTimeouts (since we're triggering idle manually).
-    // Otherwise, it will get into a bad internal state with uncaught exceptions.
-    await delay(2000);
```

Comment:
At last the magic trick has been revealed. I was fooled the entire time!

Reply:
My next trick will be a disappearing act :wink: